omnibus/files/private-chef-cookbooks/private-chef/libraries/elasticsearch_index_provider.rb

```diff
@@ -5,33 +5,70 @@ class Provider
     class ElasticSearchIndex < Chef::Provider::LWRPBase
       use_inline_resources if defined?(:use_inlined_resources)
```

Comment:
While we are modifying this substantially, it may be a good time to move this to a new-style chef resource.